submissions/ses3332002/html-css-popup/build/style.css

```
@@ -0,0 +1,144 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+  background-color: white;
+}
+body a {
+  text-decoration: none;
+  color: unset;
+}
+body a:focus {
+  outline-style: solid;
+  outline-offset: 4px;
+  outline-color: #6098f3;
+  outline-width: 2px;
+}
+body input[type=checkbox] {
+  position: absolute;
+  -webkit-appearance: none;
+  appearance: none;
+}
+
+header {
+  text-align: right;
+}
+header .user_menu {
+  display: inline-flex;
+  align-items: center;
+  justify-content: space-around;
+  font-size: 0;
+}
+header .user_menu .user_menu__element {
+  margin: 10px;
+  cursor: pointer;
+}
+header .user_menu .user_menu__element:hover {
+  outline-style: solid;
+  outline-offset: 4px;
+  outline-color: #6098f3;
+  outline-width: 2px;
+}
+header .user_menu .user_menu__element-text {
+  font-size: 14px;
+}
+header .user_menu .apps_menu__checkbox:focus + .more_apps__label {
+  outline-style: solid;
+  outline-offset: 4px;
+  outline-color: #6098f3;
+  outline-width: 2px;
```

It's not a blocker, but you can simplify your styles for this selector.
```suggestion
  outline: 2px solid #6098f3;
  outline-offset: 4px;
```